workbook/components/session_service.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+from django.utils import timezone
+
+from workbook.models.models import Session
+
+
+class SessionService:
+    def get_time_difference(self, session_start_time):
+        time_difference = timezone.now() - session_start_time
+        return time_difference.total_seconds() / 60
+
+    def is_valid(self, session_start_time):
+        return self.get_time_difference(session_start_time) < 1
+
+    def get(self, user_id):
+
+        session = Session.objects.filter(user_id=user_id).order_by('-token_start_time').first()  # '-' desc
+
+        if session and self.is_valid(session.token_start_time):
+            return session
+
+        return None
```

Comment:
this code is 100% good just an info, you use this syntax, it would save you some lines ^^
```suggestion
        return session if session and self.is_valid(session.token_start_time) else None 
```